versioned_docs/version-0.2.0/contribute-to-kubeslice-docs.md

```diff
@@ -0,0 +1,52 @@
+# Contribute to KubeSlice documentation
+
+_KubeSlice welcomes suggestions for improvement from all contributors, new and experienced!_
+
+You can contribute to the KubeSlice documentation by
+
+- Improving existing content
+- Suggesting new content
+
+## Getting Started
+
+Everybody is welcome to suggest changes by submitting a pull request
+or report a bug/typo with the help of a GitHub issue to the
+[docs](https://github.com/kubeslice/docs) repository.
+
+## Pre-requisites 
```

Comment:
We have used Prerequisites as a word without a hyphen. 

Change Pre-requisites to Prerequisites